cmd/root.go

```diff
@@ -1,20 +1,6 @@
-/*
-Copyright 2017 The Reposeed Authors.
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-    http://www.apache.org/licenses/LICENSE-2.0
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package main
+package cmd
```

Comment:
What's the reasoning behind removing the Copyright notice?

Reply:
Sorry about that. forgot to copy the copyright section.